libcrux-ml-dsa/src/simd/portable/ntt.rs

```diff
@@ -76,6 +147,9 @@ fn ntt_at_layer_0(re: &mut [Coefficients; SIMD_UNITS_IN_RING_ELEMENT]) {
         zeta_2: i32,
         zeta_3: i32,
     ) {
+        hax_lib::fstar!(
+            "reveal_opaque (`%Spec.Utils.is_i32b_array_opaque) (Spec.Utils.is_i32b_array_opaque)"
+        );
```

Comment:
Let's make sure there's some separation between proofs and coder everywhere.

```suggestion
        );

```

Reply:
Done